pallets/gear/src/tests.rs

```diff
@@ -14295,7 +14295,7 @@ fn send_gasless_message_works() {
                     keep_alive: false,
                 }
             ),
-            Error::<Test>::FailureRedeemingVoucher
+            pallet_gear_bank::Error::<Test>::InsufficientBalance
```

Comment:
```suggestion
            pallet_gear_voucher::Error::<Test>::InsufficientBalance
```

Reply:
Not possible yet, will be replaced in following PRs